splunk-otel-android/src/main/java/com/splunk/rum/BandwidthTracker.java

```diff
@@ -32,7 +32,7 @@ class BandwidthTracker {
     private final ArrayDeque<Long> sizes = new ArrayDeque<>();
 
     BandwidthTracker() {
-        this(Clock.systemDefaultZone());
+        this(Clock.systemUTC());
```

Comment:
Why does this matter? Does it prevent problems around time changes or ?

Reply:
It triggers this check: https://errorprone.info/bugpattern/JavaTimeDefaultTimeZone
Which probably does not matter at all for this project, but since the only thing we need the clock for is to get the current millis, it can be any clock, we don't care about the timezone.